Lecture-6/Task2/l6task2/Application.java

```diff
@@ -0,0 +1,95 @@
+package l6task2;
+
+import java.util.*;
+
+public class Application {
+
+    private static String[] firstNames = {"Kate", "Bob", "Roy", "Jane", "Tom", "Nick", "Ann", "Jack", "Eva", "Adam"};
+    private static String[] lastNames = {"Smith", "Jeff", "Little", "Black", "Bush", "Neal", "Owen", "Ryder", "Ford", "Gill"};
+
+    private static int getRandomAge() {
+        return (int) (Math.random() * 50 + 20);
```

Comment:
Use `ThreadLocalRandom` class to generate random numbers:
```java
return ThreadLocalRandom.current().nextInt(minValue, maxValue);
```

Reply:
done